stl/inc/format

```diff
@@ -2138,11 +2137,12 @@ _NODISCARD _OutputIt _Write_integral(
     }
 
     // long long -1 representation in binary is 64 bits + sign
-    array<char, 65> _Buffer;
-    const auto [_End, _Ec] = _STD to_chars(_Buffer.data(), _Buffer.data() + _Buffer.size(), _Value, _Base);
+    constexpr auto _Buffer_size = 65;
+    char _Buffer[_Buffer_size];

```

Comment:
if you use `_STD end` and `_STD begin` below you don't need `_Buffer_size` multiple times, and just write it explicitly in the array declaration.